src/start.ts

```diff
@@ -309,9 +309,11 @@ async function start(): Promise<void> {
 }
 
 process.on('unhandledRejection', (reason, promise) => {
-    if (reason instanceof DiscordAPIError) {
```

Comment:
Why is this even here btw?

Reply:
If a user runs a command like a setup command and the channel deletes the channel the error will be ignored.